packages/orchestrator-ui-components/src/components/WFOSteps/WFOStepStatusIcon/WFOStepStatusIcon.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { getStyles } from '../getStyles';
+import type { EuiThemeComputed } from '@elastic/eui';
+import { useOrchestratorTheme } from '../../../hooks';
+import { StepStatus } from '../../../types';
+import { WFOCogFill } from '../../../icons/WFOCogFill';
+import { WFOCheckmarkCircleFill } from '../../../icons';
+import { WFOXCircleFill } from '../../../icons';
+import { WFOMinusCircleFill } from '../../../icons';
+
+export interface WFOStepStatusIconProps {
+    stepStatus: StepStatus;
+}
+
+interface SubIconProps {
+    stepStatus: StepStatus;
+    theme: EuiThemeComputed;
+}
+
+const SubIcon = ({ stepStatus, theme }: SubIconProps) => {
+    switch (stepStatus) {
+        case StepStatus.PENDING:
+            return <></>;
+        case StepStatus.SUSPEND:
+            return <WFOMinusCircleFill color={'#8E6A00'} />;
```

Comment:
isnt this the colors.warningText? (same goes for different colors at line 46, 48 and 50 )

Reply:
Yeah so. Close a bit but not really. These colors are not in the theme. They are also quite specific to these icons so I thought it was ok to leave them here. I will ask Wouter what to do here, change the design color to standard warning or use a one of color